Userland/Libraries/LibWeb/Platform/Timer.cpp

```diff
@@ -17,7 +17,7 @@ NonnullRefPtr<Timer> Timer::create()
     return EventLoopPlugin::the().create_timer();
 }
 
-NonnullRefPtr<Timer> Timer::create_repeating(int interval_ms, Function<void()>&& timeout_handler)
+NonnullRefPtr<Timer> Timer::create_repeating(int interval_ms, JS::SafeFunction<void()>&& timeout_handler)
```

Comment:
Have you checked if there are any existing uses that manually capture handles?

Reply:
Everything currently uses Platform::Timer as if it used JS::SafeFunction (e.g. capturing raw pointers to this instead of using handles) :sweat_smile:

Reply:
:yaksplode: